Automattic/wc-calypso-bridge
# Remove WCPay menu

fixes: woocommerce/woocommerce#32204

This PR removes the WCPay menu/experiment and its dependencies but keeps the typescript support.

**Testing instructions**

1. Deactivate & uninstall WooCommerce Payments plugin
2. Load up this branch and observe everything still works and no payment menu on the page.

**Notes**
**We should merge this once we're ready to release WCPayments menu in WCA**
